packages/compose/src/hooks/use-disabled/index.ts

```diff
@@ -39,11 +39,16 @@ export default function useDisabled( {
 				return;
 			}
 
+			const defaultView = node?.ownerDocument?.defaultView;
```

Comment:
Could this ever be empty?

Reply:
I don't know in practical cases, but TypeScript will output errors if this check is not present.44

![falsy](https://user-images.githubusercontent.com/54422211/214853853-36abf3bf-c618-41c0-8da6-b245102de0db.png)